packs/data/feats.db/munitions-crafter.json

```diff
@@ -51,6 +51,30 @@
             {
                 "key": "GrantItem",
                 "uuid": "Compendium.pf2e.feats-srd.Alchemical Crafting"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "system.crafting.entries.munitionsCrafter.fieldDiscovery",
```

Comment:
I don't think there's a need to have these all be AEs.

The ones for alchemist only do that because the choice of Field Discovery will update the entry. You could set all of these values in the Crafty Entry RE and just update the Crafting Entry RE to use them if needed

Reply:
That is a good point, makes it easier to get an overview as well